great_expectations_cloud/agent/agent.py

```diff
@@ -14,7 +14,10 @@
 import orjson
 from great_expectations import get_context  # type: ignore[attr-defined] # TODO: fix this
 from great_expectations.compatibility import pydantic
-from great_expectations.compatibility.pydantic import AmqpDsn, AnyUrl
+from great_expectations.compatibility.pydantic import (  # noqa: TCH002  # cannot be imported in type checking block
+    AmqpDsn,
+    AnyUrl,
```

Comment:
This needs to be updated due to the model name change.

https://github.com/great-expectations/cloud/blob/78ae39de7968ba8eb878a48bdd65e7fb7be84d8d/pyproject.toml#L173-L181